src/services/objections.api.service.ts

```diff
@@ -1,5 +1,22 @@
 import logger from "../utils/logger";
 
+/**
+ * State of Objection
+ */
+export enum ObjectionStatus {
```

Comment:
I'm keeping everything in the same module for the time being, should be easier to migrate to the sdk in future.